commandline/src/main/java/tech/pegasys/web3signer/commandline/Web3SignerCommand.java

```diff
@@ -172,6 +172,14 @@
       arity = "1")
   private final FilecoinNetwork filecoinNetwork = FilecoinNetwork.TESTNET;
 
+  @Option(
+      names = {"--slashing-protection-enabled"},
+      description =
```

Comment:
it's a good idea to use `paramLabel=<BOOL>`. Otherwise picocli prints variable name.

Reply:
done